### DISCUSSION TITLE
Serve a Nuxt 3 app inside a unjs/Citty CLI

### DISCUSSION THREAD

**Discussion by Met96** [nuxt/nuxt#23662] [General]
Hi guys,
I need to know if there is a possibility of serving a Nuxt 3 app published within a Citty CLI.

My project is a multirepo, composed with:
- /app: nuxt 3 apps to serve
- /cli: a CLI with unjs/Citty.

My real idea is to build Nuxt app within the CLI, publish it and finally run a command like:
PORT=4000 node [GLOBAL CLI PATH INSTALLED INSIDE COMPUTER]/app

But I have some problems with the dependencies imported within the app or with server api.

Can you give me more ideas or possibilities to make this real?
Than you so much
Mattia

**Comment by Rigo-m** [comment]
@danielroe could you lend us a hand here? It'd be cool to be able to bundle a runnable npm package with a nuxt 3 app inside it
